kratos/processes/structured_mesh_generator_process.h

```diff
@@ -139,7 +136,10 @@ namespace Kratos
 		  std::size_t mConditiongPropertiesId;
 		  std::string mElementName;
 		  std::string mConditionName;
+          std::string mBodySubModelPartName;
+          std::string mSkinSubModelPartName;
           bool mCreateSkinSubModelPart;
+          bool mCreateBodySubModelPart;
 		  ModelPart& mrOutputModelPart;
```

Comment:
Fix tabulations

Reply:
The ones I added follow our standard, the ones already there are the wrong ones. I didn't fix them to avoid refactor changes. I can make them consistent though.

Reply:
Please

Reply:
Now the header follows our formatting convention. Note that I also took the chance to add some documentation.